package.xml

```diff
@@ -62,54 +62,44 @@
     <license uri="https://github.com/DataDog/dd-trace-php/blob/master/LICENSE">BSD 3-Clause</license>
     <notes>
         <![CDATA[
-This release adds support for PHP 8.3 for the tracer and the profiler. Please be aware that we are currently building against PHP 8.3.0 RC3.
+This release adds support to OpenTelemetry so that you can use the OpenTelemetry API to generate Datadog spans. This is off by default. Enable it with the environment variable `DD_TRACE_OTEL_ENABLED=1` or INI setting `datadog.trace.otel_enabled=1. Also, the tracer now generates 128 bits trace ids by default, in an effort to match OpenTelemetry standards.
 
 ## Tracer
 ### Added
-- Add Magento 2 Integration #2274
-- Support for PHP 8.3 #2098, #2282
-- Add a configuration flag to disable laravel queue distributed tracing #2304
-- Laminas RESTful methods handling #2322
-- Expand arrays in meta and metrics into nested-tag format #2302, #2333
-- Report extensions used in telemetry #2295
+- Set _dd.base_service when overriding service name for a span #2275
+- OpenTelemetry API #2332
+- Add tracing for curl_multi_exec() #2347
+- CakePHP http.route implementation #2360
 
 ### Changed
-- Redis Span Removal and Priority Sampling Orphans Cleanup #2306
-- Subclass SpanData with RootSpanData #2309
-- Refactor tightly coupled extraction state #2314
+- Enable 128-bit trace ID generation by default #2358, #2368
+- Respect DD_LOGS_INJECTION default value #2365
+- Completely avoid instrumenting the PSR NullLogger #2367
+- Inject the _dd.dbm_trace_injected tag only when full propagation is enabled #2346
+- Fix PHP5: The Magento Integration is not part of PHP5 #2363
+- Fix orphans removal when 128-bit is enabled #2366
+- Use _dd.agent_psr instead of _dd.rule_psr for agent sampling #2370
```

Comment:
Now you moved somehow these from fixed to changed

Reply:
Wait, isn't it what you wanted?

Reply:
w/e I'll move them back :+1:

Reply:
When you add a comment in github to one line, it'll show you the three preceding lines as well.
When you select two or more lines, it just displays these (and will show "comment on lines X to Y"). That's probably a bit confusing :-D

Reply:
It definitely is